actions/send.server.info.email.yaml

```diff
@@ -1,98 +1,116 @@
 ---
-description: Email notifications for error emails
+description: Sends email notifications to VM owners based on a given query.
 enabled: true
 entry_point: workflows/send.server.info.email.yaml
```

Comment:
I'm going to bikeshed here a little

Send server info implies it's just a daily report. Instead it would be better if we called it something like `report.server.problems`. Then leave the fact it's by email as an implementation detail, since its equally valid to use Jira / Slack / Letters in the post....etc.